CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ customers cannot upgrade their bootloader, its changes are recorded separately.
 ## Firmware
 
 ### [Unreleased]
+
+### 9.8.0 [released 2021-10-20]
```

Comment:
is it actually released? or are you going to make a release today?

Reply:
I was hoping we could make it today. I reverted that part and will update it once the release actually happens.